sqlalchemy_drill/drilldbapi/_drilldbapi.py

```diff
@@ -10,15 +10,17 @@
 from datetime import date, time, datetime
 from time import gmtime
 from . import api_globals
-from .api_exceptions import AuthError, DatabaseError, ProgrammingError, \
-    CursorClosedException, ConnectionClosedException
+from .api_exceptions import (AuthError, ConnectionClosedException, CursorClosedException,
+                             DatabaseError, Error, IntegrityError, InterfaceError, InternalError,
+                             NotSupportedError, OperationalError, ProgrammingError, Warning)
 
 apilevel = '2.0'
 threadsafety = 3
 paramstyle = 'qmark'
 default_storage_plugin = ''
 
 logger = logging.getLogger('drilldbapi')
+logger.setLevel(logging.DEBUG)
```

Comment:
Do we want to leave this with the `DEBUG` level?

Reply:
Oh good catch thanks.